README.md

```diff
@@ -30,6 +30,20 @@ For a demo, visit [https://openlayers-editor.geops.de](https://openlayers-editor
 - Run: `npm start`
 - Open your browser and visit [http://localhost:5000](http://localhost:5000)
 
+## Docker & docker-compose
+Build and run locally (image not available on Docker Hub yet!)
```

Comment:
I agree that a public ready-to-use docker image would be nicer. I'll be happy to create a dedicated image repo on the Docker Hub if this gets merged.